app/src/main/res/values/strings.xml

```diff
@@ -46,7 +46,7 @@
     <string name="clipboard_timeout_summary">Time copied usernames or passwords are stored in clipboard.</string>
     <string name="clipboard_swipe_clean">Swipe to clear clipboard now</string>
     <string name="select_to_copy">Select to copy %1$s to clipboard</string>
-    <string name="creating_db_key">Creating database key…</string>
+    <string name="creating_db_key">Creating database keyfile…</string>
```

Comment:
It's not a keyfile, it's the global encryption key recover to unlock the database